releases/evm/2025-02-XRPL-EVM-GMP-v6.0.4.md

```diff
@@ -0,0 +1,236 @@
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Devnet-amplifier** | `0xba76c6980428A0b10CFC5d8ccb61949677A61233`                                                                                                                                           |
+| **Stagenet**         | `0xba76c6980428A0b10CFC5d8ccb61949677A61233`                                                                                                                                           |
+| **Testnet**          | `0xba76c6980428A0b10CFC5d8ccb61949677A61233`                                                                                                                                           |
+| **Mainnet**          | `0xB8Cd93C83A974649D76B1c19f311f639e62272BC`, `0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05`, `0x5b593E7b1725dc6FcbbFe80b2415B19153F94A85`, `0xE86375704CDb8491a5Ed82D90DceCE02Ee0ac25F` |
+
+2. Deploy `ConstAddrDeployer`:
+
+- `stagenet` and `testnet` use the same contract address, so we only deploy on `testnet`.
+
+| Network              | `deployer address`                           |
+| -------------------- | -------------------------------------------- |
+| **Devnet-amplifier** | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
+| **Stagenet**         | X                                            |
+| **Testnet**          | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
+| **Mainnet**          | `0xE86375704CDb8491a5Ed82D90DceCE02Ee0ac25F` |
+
+```bash
+node evm/deploy-contract.js -c ConstAddressDeployer -m create --artifactPath ../evm/legacy/ConstAddressDeployer.json
+```
+
+3. Deploy `Create3Deployer`:
+
+- `stagenet` and `testnet` use the same contract address, so we only deploy on `testnet`.
+
+| Network              | `deployer address`                           |
+| -------------------- | -------------------------------------------- |
+| **Devnet-amplifier** | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
+| **Stagenet**         | X                                            |
+| **Testnet**          | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
+| **Mainnet**          | `0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05` |
+
+```bash
+node evm/deploy-contract.js -c Create3Deployer -m create2
+```
+
+4. Waste nonce, this step should only be performed on `testnet` and `mainnet`. To generate the same `AmplifierGateway` address as older EVM chains we need to waste 2 nonce on the deployer key.
+
+```bash
+node evm/send-tokens.js -r 0xba76c6980428A0b10CFC5d8ccb61949677A61233 --amount 0.0001 # burn nonce 0
+node evm/send-tokens.js -r 0xba76c6980428A0b10CFC5d8ccb61949677A61233 --amount 0.0001 # burn nonce 1
+```
+
+5. Deploy Gateway contract
```

Comment:
I feel like we should just set this as a global param on the config and default to this for each environment. Maybe we can log a ticket for this and pass to someone.